packages/api-fetch/README.md

```diff
@@ -54,6 +54,10 @@ Unlike `fetch`, the `Promise` return value of `apiFetch` will resolve to the par
 
 Shorthand to be used in place of `body`, accepts an object value to be stringified to JSON.
 
+#### `responseBody` (`object`)
+
+If specified, the default fetch handler will short-circuit and return the value of this option as a response. This is especially useful in middlewares.
+
```

Comment:
Don't think I've ever seen this in a middleware implementation. Why can't the middleware simply _not_ call the next middleware?

Reply:
Agreed. I think it's possible?

Reply:
> Why can't the middleware simply not call the next middleware?

Good spot Mr socks, we don't have to get to the end of the chain. Updating now.

> Don't think I've ever seen this in a middleware implementation. 

How could have you when it's only proposed now :D